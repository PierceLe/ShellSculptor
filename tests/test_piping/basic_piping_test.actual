>> echo "Testing is so boring" | cat
Testing is so boring
>> cat ~/test_data/movies.txt | grep "Advengers"
cat: /Users/piercele/test_data/movies.txt: No such file or directory
>> cat ~/test_data/football.csv | grep "Manchester United"
cat: /Users/piercele/test_data/football.csv: No such file or directory
>> cat ~/test_data/football.csv | grep "2-2" | wc -l
cat: /Users/piercele/test_data/football.csv: No such file or directory
       0
>> cat ~/test_data/football.csv | grep "0-" | awk -F',' '{print $1}'
cat: /Users/piercele/test_data/football.csv: No such file or directory
>> cat ~/test_data/football.csv | sort -t',' -k2
cat: /Users/piercele/test_data/football.csv: No such file or directory
>> cat ~/test_data/football.csv | awk -F',' '{print $2}'
cat: /Users/piercele/test_data/football.csv: No such file or directory

