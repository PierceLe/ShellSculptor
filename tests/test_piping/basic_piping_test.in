echo "Testing is so boring" | cat
cat ~/test_data/movies.txt | grep "Advengers"
cat ~/test_data/football.csv | grep "Manchester United"
cat ~/test_data/football.csv | grep "2-2" | wc -l
cat ~/test_data/football.csv | grep "0-" | awk -F',' '{print $1}'
cat ~/test_data/football.csv | sort -t',' -k2
cat ~/test_data/football.csv | awk -F',' '{print $2}'
