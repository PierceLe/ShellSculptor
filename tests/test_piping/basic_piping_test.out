>> echo "Testing is so boring" | cat
Testing is so boring
>> cat ~/test_data/movies.txt | grep "Advengers"
2. Advengers: Endgame
4. Advengers: Infinity wars
>> cat ~/test_data/football.csv | grep "Manchester United"
Swansea City,         1-4, 0-2, Manchester United
>> cat ~/test_data/football.csv | grep "2-2" | wc -l
1
>> cat ~/test_data/football.csv | grep "0-" | awk -F',' '{print $1}'
Norwich City
Sunderland
Swansea City
West Bromwich Albion
Crystal Palace
>> cat ~/test_data/football.csv | sort -t',' -k2
Sunderland,           0-1, 0-0, Fulham
West Bromwich Albion, 0-1, 0-0, Southampton
Crystal Palace,       0-1, 0-0, Tottenham Hotspur
Liverpool,            1-0, 1-0, Stoke City
Arsenal,              1-3, 1-1, Aston Villa
Swansea City,         1-4, 0-2, Manchester United
West Ham United,      2-0, 1-0, Cardiff City
Chelsea,              2-0, 2-0, Hull City
Norwich City,         2-2, 0-0, Everton
Manchester City,      4-0, 2-0, Newcastle United 
Team1,                FT,  HT,  Team2
>> cat ~/test_data/football.csv | awk -F',' '{print $2}'
                FT
              1-3
            1-0
         2-2
           0-1
         1-4
 0-1
      2-0
              2-0
       0-1
      4-0

